### DISCUSSION TITLE
Content not updating when using Custom domain name

### DISCUSSION THREAD

**Discussion by tbabyuk** [vercel/community#3606] [Help] [answered]
### Summary

Hello community,

A question if I may. I recently created a new project from GitHub (a Next.js app) and everything worked well. If I click on the automatically-generated url, I get the right content and can see my app no problem. However, I wanted to have a custom domain name for this app, which I pointed to Vercel and Vercel says the domain is correctly connected up. But after I try to visit my project using this custom domain, I keep getting my old website (the one I had earlier under this domain instead of the new content). 

I know there is something called propagation but it has already been over a day and I still keep getting the same thing. I also tried clearing cache and hard refreshing - nothing. Could anyone land a hand?

Thank you!

### Steps to Reproduce

n/a

**Comment by HMJosh** [comment]
Is there any chance you still have a dns record pointing to the old one?

**Comment by amyegan** [comment]
Have you created a new deployment or redeployed after adding the custom domain?

**Reply by klovistore2** [reply]
yes!

**Comment by klovistore2** [comment]
same issue here, if I go to mydomain.com I don't have the same behavior than mylastdeployment.vercel.app !

**Comment by pawankushwah** [comment]
you just need to Hard Reload the website if your website is loading static content. I had a same issue and when i hard reloaded the website, It worked.

**Comment by amyegan** [comment, marked as answer]
Thanks for the suggestions, everyone! I'm combining them into a single post.
- Create a new deployment after adding the custom domain - https://github.com/vercel/community/discussions/3606#discussioncomment-6730476

- Hard refresh the website to reset the results in your browser - https://github.com/vercel/community/discussions/3606#discussioncomment-10073470

- Double check DNS records - https://github.com/vercel/community/discussions/3606#discussioncomment-6722810

[a comment by an automated account is not shown]
